src/components/quiz-review/question-list-review/question-list-review.tsx

```diff
@@ -24,13 +24,13 @@ export interface QuestionListReviewProps {
 
 const translates = (): QuizTranslates => {
   return {
-    quizScore: <Text id="ivq.quiz_score">Your score is</Text>,
+    quizScore: <Text id="ivq.quiz_score" data-testid="quizScoreTitle">Your score is</Text>,
     retakeButton: <Text id="ivq.retake_button">Retake</Text>,
-    retakeButtonAreaLabel: <Text id="ivq.retake_button_area_label">Click to retake the quiz</Text>,
+    retakeButtonAreaLabel: <Text id="ivq.retake_button_area_label" data-testid="reviewRetakeButton">Click to retake the quiz</Text>,
```

Comment:
<Text> tag transforms into plain text by i18n library so you can't add `data-testid` attribute to the <Text> tag. `data-testid` attribute should be added to parent element, like you did here:
https://github.com/kaltura/playkit-js-ivq/pull/42/files#diff-c9b14caf05ca3762b3ff7f7ba9b055fe203f5f3e1417396cc16a67d2d61a255aR106
Result in the DOM:
<img width="831" alt="image" src="https://user-images.githubusercontent.com/51074448/180969964-b75bf43f-0d43-417e-83fa-4d3e94ed1550.png">